src/components/project/project-member/project-member.service.ts

```diff
@@ -44,49 +33,15 @@ import { ProjectMemberRepository } from './project-member.repository';
 @Injectable()
 export class ProjectMemberService {
   constructor(
-    private readonly db: DatabaseService,
-    private readonly config: ConfigService,
     @Inject(forwardRef(() => UserService))
     private readonly userService: UserService & {},
-    private readonly eventBus: IEventBus,
     @Inject(forwardRef(() => ProjectService))
     private readonly projectService: ProjectService & {},
     @Logger('project:member:service') private readonly logger: ILogger,
     private readonly privileges: Privileges,
     private readonly repo: ProjectMemberRepository,
   ) {}
 
-  protected async verifyRelationshipEligibility(
-    projectId: ID,
-    userId: ID,
-  ): Promise<void> {
-    const result = await this.repo.verifyRelationshipEligibility(
-      projectId,
-      userId,
-    );
-
-    if (!result?.project) {
-      throw new NotFoundException(
-        'Could not find project',
-        'projectMember.projectId',
-      );
-    }
-
-    if (!result?.user) {
-      throw new NotFoundException(
-        'Could not find person',
-        'projectMember.userId',
-      );
-    }
-
-    if (result.member) {
-      throw new DuplicateException(
-        'projectMember.userId',
-        'Person is already a member of this project',
-      );
-    }
```

Comment:
may need to move these checks to the verifyRelationshipEligibility function in the project-member.repository.ts file